diff -Nru  ../orig/qemu-9.0.4/tests/qtest/bios-tables-test.c tests/qtest/bios-tables-test.c
--- ../orig/qemu-9.0.4/tests/qtest/bios-tables-test.c   2025-10-18 10:54:02.288118600 -0400
+++ tests/qtest/bios-tables-test.c      2025-10-18 10:54:21.058214500 -0400
@@ -106,7 +106,7 @@
 static char disk[] = "tests/acpi-test-disk-XXXXXX";
 static const char *data_dir = "tests/data/acpi";
 #ifdef CONFIG_IASL
-static const char *iasl = CONFIG_IASL;
+static const char *iasl = "CONFIG_IASL";
 #else
 static const char *iasl;
 #endif
